### PR TITLE
feat: osnap execution warning message

### PR DESCRIPTION
**Pull request by gsteenkamp89**
## motivation

We want to show oSnap users a warning if their safe is misconfigured and their transaction will not auto execute.
We call an endpoint at `https://osnap.uma.xyz/api/sapce-config` to inspect the **on-chain** settings for a given safe.

In Proposals list (already proposed)
Screenshot 2024-02-19 at 18 36 35
Screenshot 2024-02-19 at 18 36 51

In transaction Builder  (before proposal)
Screenshot 2024-02-19 at 18 41 16

### CODE REVIEW

File: src/plugins/oSnap/Proposal.vue [hunk 1]
```diff
@@ -59,9 +60,11 @@ function enrichTransactionForDisplay(transaction: Transaction) {
...
       type: 'Contract interaction',
       'method name': method.name,
-      ...Object.fromEntries(method.inputs.map((input,i)=>{
-        return [`${input.name} (param ${i+1}): `,parameters[i]]
-      }))
+      ...Object.fromEntries(
+        method.inputs.map((input, i) => {
+          return [`${input.name} (param ${i + 1}): `, parameters[i]];
+        })
+      )
```
Comment (gsteenkamp89) [inline src/plugins/oSnap/Proposal.vue:67]: formatting changes only

**Comment by daywiss** [comment]
seeing an issue on space: https://snapshot-77xkgjvgj-uma.vercel.app/#/umadev.eth/create
image
image

not sure if this is us or snapshot

**Review by daywiss — APPROVED** [review]
i think we are good to go on this. ill pr it to snapshot

**Comment by daywiss** [comment]
prd here: https://github.com/snapshot-labs/snapshot/pull/4567

**Pull request merged**